tests/test_api.py

```diff
@@ -327,7 +327,7 @@ def test_basic(self, config):
             keras_model = build_fn(
                 meta=meta,
                 hidden_layer_sizes=(100,),
-                compile_kwargs={"optimizer": "adam", "loss": None, "metrics": None,},
+                compile_kwargs={"optimizer": "adam", "loss": "auto", "metrics": None,},
```

Comment:
These tests are directly calling the model building function which is debatably not a great way to set up tests. This just changes the inputs to the model building function to reflect what it would now be getting if users don't specify a loss function.